dist/stat_data.py

```diff
@@ -113,12 +113,15 @@ def __init__(self, name, desc, flags=''):
     ConnStat('cond_auto_wait_reset', 'auto adjusting condition resets'),
     ConnStat('cond_wait', 'pthread mutex condition wait calls'),
     ConnStat('file_open', 'files currently open', 'no_clear,no_scale'),
+    ConnStat('fsync_active', 'active filesystem fsync calls','no_clear,no_scale'),
```

Comment:
Thanks!  Looking at the code, we probably want to be consistent and add in a total fsync counter, similar to `read_io` and `write_io`.  Thinking more generally, those 6 stats, plus `file_open` seem to make a critical mass to have a new section of stats, `FSStat` for file system stats.  @agorrod  is that worthwhile?  Are there other stats on the horizon that might fit in with file system ops for keeping track of things with pluggable file systems?